src/ADAL.PCL.iOS/Logger.cs

```diff
@@ -35,28 +35,40 @@ internal class Logger : LoggerBase
         internal override void Error(CallState callState, Exception ex, [System.Runtime.CompilerServices.CallerFilePath] string callerFilePath = "")
         {
             string message = PrepareLogMessage(callState, GetCallerFilename(callerFilePath), ex.ToString());
-            Console.WriteLine(message); //Console.writeline writes to NSLog by default
+            if (LoggerCallbackHandler.DefaultLogging)
+            {
+                Console.WriteLine(message); //Console.writeline writes to NSLog by default
+            }
```

Comment:
line break. please scrub through the code to make check there are no other repeats of this.